hackclub.com.yaml

```diff
@@ -2686,4 +2686,9 @@ internal:
 - ttl: 1
   type: A
   value: 134.209.190.125
+
+node:
+- ttl: 1
+  type: CNAME
+  value: node.dino.icu.
```

Comment:
This doesn't appear to resolve. Where is your site hosted?

Reply:
My site is hosted on github only but i want to have a dino.icu domain

Reply:
Alright, then this block needs to be moved into the dino.icu.yml file, and the value needs to point to your GitHub pages site.

Reply:
Alright, made the changes